collection-scripts/monitoring_common.sh

```diff
@@ -34,8 +34,8 @@ metrics_get() {
     -c prometheus \
     -n openshift-monitoring \
     -- promtool tsdb dump-openmetrics /prometheus --sandbox-dir-root="/prometheus" "$@" \
-       > "$METRICS_PATH/metrics.openmetrics" \
-       2> "$METRICS_PATH/metrics.stderr"
+      | gzip > "$METRICS_PATH/metrics.openmetrics.gz" \
```

Comment:
I assume that this will be maintained by monitoring team. Did they acknowledge this change?. How do we know that this won't break anywhere?